third_party/accessibility/ax/platform/ax_platform_node_mac.mm

```diff
@@ -1016,8 +1016,10 @@ - (NSRange)accessibilityRangeForLine:(NSInteger)line {
 }
 
 - (NSRange)accessibilityRangeForPosition:(NSPoint)point {
-  BASE_UNREACHABLE();
-  return NSMakeRange(0, 0);
+  if (!_node)
+    return NSMakeRange(0, 0);
+
+  return NSMakeRange(0, [self accessibilityNumberOfCharacters]);
```

Comment:
Why the total length? do you know why this is called?

Reply:
The behavior is indifferent whether you return `(0, 0)` or `(0, [self accessibilityNumberOfCharacters])`. The goal of this PR is to throw out `BASE_UNREACHABLE()` so macos apps don't crash.

Reply:
This is called when the user triggers an a11y zoom, which shows something similar to a huge tooltip of the text under the cursor. So for example if you hover over an `NSTextField` (AppKit label/textfield), it'll take the point (in window coords) and do a hit test to figure out which UI element is at that point, then call this method to get a range for the text to display.

For widgets with very long strings of text (e.g. the text area in a Google Doc) the accessibility system may want to break this up and only return a subrange around the point instead of the full value of the widget. We could do that later if it becomes an issue.

Reply:
according to the doc, it should return the range of glyph
https://developer.apple.com/documentation/appkit/nsaccessibility/1531615-accessibilityrangeforposition
Let's at least try to do the right thing even if it is indifferent

Reply:
Oh interesting; if that's the case then let's see if we can hit-test this for the character in question.

Reply:
@cbracken is hit-testing a specific character possible? The AXTree does not have access to the RenderObjectTree; hence, we cannot translate the NSPoint(x, y) to any particular position in Text/Strings rendered by flutter. If that's true, then we will be unable to implement this function in any meaningful way. For now, taking out `BASE_UNREACHABLE` will be the appropriate fix.